shared/logutil/logutil.go

```diff
@@ -28,3 +30,26 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+//Masks the url credentials before logging for security purpose
+//[scheme:][//[userinfo@]host][/]path[?query][#fragment] -->  [scheme:][//[***]host][/***][#***]
+//if the format is not matched nothing is done, string is returned as is.
+func MaskCredentialsLogging( currUrl string) string {
+	MaskedUrl := currUrl
+	u, err := url.Parse(currUrl)
+	if err != nil {
+		logrus.Info("Not a valid URL format to mask")
```

Comment:
This might be a noisy log. Do you think it should be a DEBUG level log? I think we should remove it.
```suggestion
```